runtime/parser/expression.go

```diff
@@ -1726,6 +1726,9 @@ func parseIntegerLiteral(p *parser, literal, text []byte, kind IntegerLiteralKin
 		if withoutUnderscores == "" {
 			report(InvalidNumberLiteralKindMissingDigits)
 		} else {
+			estimatedSize := common.OverEstimateBigIntFromString(withoutUnderscores)
+			common.UseMemory(p.memoryGauge, common.NewBigIntMemoryUsage(estimatedSize))
```

Comment:
now that the bigInt is metered here, we can remove the metering from the interpreter:
https://github.com/onflow/cadence/blob/ab15f9a9bea9a7e71502514df3d54553c70c1fb5/runtime/interpreter/interpreter_expression.go#L560-L565

Reply:
not for all of them, but for the ones that uses the bigInt as is.
i.e: `Int`, `UIntType`, `Int128Type `, `Int256Type`, `UInt128Type `, `UInt256Type`,  `Integer`, `SignedInteger`

Reply:
Removed in https://github.com/onflow/cadence/pull/2110/commits/4cecda419e9b624a0dad3dc4f84913b3c23d5d3e